docs/whats-new.md

```diff
@@ -2,6 +2,31 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.5 (In development)
+
+### MVT Layer
+
+#### Binary mode
```

Comment:
Move this to upgrade-guide.md

Reply:
I take it you just mean this one section (`Binary mode`), and not the other two (`Direct binary parsing` and `Polygon triangulation in worker`)

Reply:
I've moved this now